src/components/AmountFilter.jsx

```diff
@@ -11,20 +11,34 @@ export default function AmountFilter(props) {
   return (
     <Grid container>
       <Grid item sx={{ m: 1 }}>
-        <InputLabel sx={{ pl: 1 }}>Min Amount</InputLabel>
+        <InputLabel sx={{ pl: 1 }}>Min</InputLabel>
         <AmountTextField
           error={error}
           value={min || ''}
-          onChange={onMinChange}
+          onChange={(e) => onMinChange(e.target.value)}
         />
       </Grid>
       <Grid item sx={{ pt: 2, display: 'flex', alignItems: 'center' }}>
         <RemoveIcon />
       </Grid>
       <Grid item sx={{ m: 1 }}>
-        <InputLabel sx={{ pl: 1 }}>Max Amount</InputLabel>
-        <AmountTextField value={max || ''} onChange={onMaxChange} />
+        <InputLabel sx={{ pl: 1 }}>Max</InputLabel>
+        <AmountTextField
+          value={max || ''}
+          onChange={(e) => onMaxChange(e.target.value)}
+        />
       </Grid>
+
+      <Slider
+        value={[min, max]}
+        min={0}
+        max={10000}
+        step={100}
+        onChange={(e) => {
+          onMinChange(e.target.value[0]);
+          onMaxChange(e.target.value[1]);
```

Comment:
I'm noticing that if you drag this slider around the page becomes slow because it's always updating the URL and trying to refresh the scholarship list. Maybe before we add the slider we should implement the **Apply Filters** button at the bottom of the filter panel?

Reply:
I'll hide it behind an experiement guard